### ISSUE TITLE
Docker installation documentation asks to add the same value to taiga-back two times in a row

### ISSUE THREAD

**Issue by nuokh** [taigaio/taiga-doc#4]
Under the section "Session Settings" on [this page](https://docs.taiga.io/setup-production.html#setup-prod-with-docker) users are getting told to add 

`
SESSION_COOKIE_SECURE: "False"
CSRF_COOKIE_SECURE: "False"
`

to "**Service: taiga-back**" two times in a row. Maybe this is a typo and it's supposed to be added to taiga-back plus a different service?

**Comment by yamila-moreno** [comment]
Hi @nuokh these are different variables, SESSION and CSRF

**Comment by nuokh** [comment]
Yes, but currently the docs tell to add the same two (different) variables to taiga-back twice.

![grafik](https://user-images.githubusercontent.com/9081831/119942252-e3539180-bf91-11eb-9bad-b89d445a8ea1.png)

**Comment by yamila-moreno** [comment]
AHH! I see! Thanks for noticing, and for the sketch!! :P

**Comment by nuokh** [comment]
My pleasure! :-)

**Comment by IK-IT-PM** [comment]
Hi, I searched in the latest documentation, and it seems that this issue has already been fixed. I believe it can be closed.
